docs/workflows/sdk-setup.mdx

```diff
@@ -22,277 +66,324 @@ The Embedded SDK is a holistic SDK solution offering the entire experience embed
 
 ## Sample App
 
-<Tabs groupId="sdks">
+<Tabs groupId="sdks" queryString="sdks">
```

Comment:
This also works without the string
```suggestion
<Tabs groupId="sdks" queryString>
```

Reply:
Interesting! I'll test locally and remove